mp_cnn/model.py

```diff
@@ -22,20 +22,20 @@ def __init__(self, n_word_dim, n_holistic_filters, n_per_dim_filters, filter_wid
                 continue
 
             holistic_conv_layers.append(nn.Sequential(
-                nn.Conv1d(n_word_dim, n_holistic_filters, ws),
+                nn.Conv1d(self.n_word_dim, n_holistic_filters, ws),
```

Comment:
Also add `self` for other parameters such as `n_holistic_filters`.

Reply:
`n_holistic_filters` is a parameter of the function whereas `n_word_dim` is no longer. `n_holistic_filters` is the same as `self.n_holistic_filters` but more concise.